docs/src/validator/get-started/setup-a-validator.md

```diff
@@ -0,0 +1,476 @@
+
+### Solana Validators
+
+After you have verified that your validator is in gossip, you can verify that your validator has joined the network using the `solana validators` command. The command lists all validators in the network, but like before, we can `grep` the output for the validator we care about:
+
+```
+solana validators | grep <pubkey>
+```
+
+You should see a line of output that looks like this:
+
+```
+5D1fNXzvv5NjV1ysLjirC4WY92RNsVH18vjmcszZd8on  FX6NNbS5GHc2kuzgTZetup6GZX6ReaWyki8Z8jC7rbNG  100%  197434166 (  0)  197434133 (  0)   2.11%   323614  1.14.17   2450110.588302720 SOL (1.74%)
+```
+
+### Solana Catchup
+
+The `solana catchup` command is a useful tool for seeing how quickly your validator is processing blocks. The Solana network has the capability to produce many transactions per second. Since your validator is new to the network, it has to ask another validator (listed as a `--known-validator` in your startup script) for a recent snapshot of the ledger. By the time you receive the snapshot, you may already be behind the network. Many transactions may have been processed and finalized in that time. In order for your validator to participate in consensus, it must _catchup_ to the rest of the network by asking for the more recent transactions that it does not have.
+
+The `solana catchup` command is a tool that tells you how far behind the network your validator is and how quickly you are catching up:
+
+```
+solana catchup <pubkey>
+```
+
+If you see a message about trying to connect, your validator may not be part of the network yet. Make sure to check the logs and double check `solana gossip` and `solana validators` to make sure your validator is running properly.
+
+Once you are happy that the validator can start up without errors, the next step is to create a system service to run the `validator.sh` file automatically. Stop the currently running validator by pressing `CTRL+C` in the window where `validator.sh` is running.
+
+## Create a System Service
+
+Follow these instructions for [running the validator as a system service](../../running-validator/validator-start#systemd-unit)
+
+Make sure to implement log rotate as well. Once you have the system service configured, start your validator using the newly configured service:
+
+```
+sudo systemctl enable --now sol
+```
+
+Now verify that the validator is running properly by tailing the logs and using the commands mentioned earlier to check gossip and Solana validators:
+
+```
+tail -f /home/sol/solana-validator*.log
+```
+
+## Monitoring
+
+`solana-watchtower` is a command you can run on a separate machine to monitor your server. You can read more about handling [automatic restarts and monitoring](../best-practices/monitoring.md#solana-watchtower) using Solana Watchtower here in the docs.
+
+## Common issues
+
+### Out of disk space
+
+Make sure your ledger is on drive with at least `1TB` of space.
```

Comment:
In the "hard drive setup" portion of this doc, it lists the min hard drive size of 2TB. Should this value also be 2TB?

```suggestion
Make sure your ledger is on drive with at least `2TB` of space.
```